README.md

```diff
@@ -0,0 +1,210 @@
+<a name="readme-top"></a>
+<div align="center">
+ 
+ 
+
+  <h3><b>Hello Microvers Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+ - [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello_Microvers] <a name="about-project"></a>
+
+>
+
+**[Hello Microvers]**  This project was part of Microverse week one tasks.
+It gives the student an opportunity to learn and do with HTML, CSS, and Github.
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+- Linters
+### Tech Stack <a name="tech-stack"></a>
+
+> The tech stack used in this project include HTML,CSS and Github.
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> This project contain the features listed below.
+
+- **[HTML]**
+- **[CSS]**
+- **[Github]**
+- **[Linters]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+doesn't require installation.
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+Open index.html with a browser
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+For now no test is implemented.
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+Copying the project folder's contents to the root of 
+your server.
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
```

Comment:
- [ ] Please notice the section in your readme which start with `>` are directions/guidelines from microverse to help you understand what to write in that section and should not be part of your readme file. Please consider removing this and all the like contents from the readme file.